code/__DEFINES/dcs/signals/signals.dm

```diff
@@ -791,6 +791,10 @@
 
 // /obj/vehicle/sealed/mecha signals
 
+#define COMSIG_KB_VEHICLE_NEW_OCCUPANT "vehicle_new_occupant"
+
+#define COMSIG_KB_VEHICLE_OCCUPANT_LEFT "vehicle_occupant_left"
+
```

Comment:
why is this comsig_kb if its not a keybind?

Reply:
removed